horreum-backend/src/main/java/io/hyperfoil/tools/horreum/svc/TestServiceImpl.java

```diff
@@ -356,9 +363,9 @@ public TestListing summary(String roles, String folder) {
           summaryList.forEach(summary -> summary.watching = subscriptionMap.computeIfAbsent(summary.id, k -> Collections.emptySet()));
       }
 
-
       TestListing listing = new TestListing();
       listing.tests = summaryList;
+      listing.count = (long)count;
```

Comment:
I think this is using server side paging (instead of paging in the db) so that `listing.count` can be created from the db response.

Reply:
Do you have a query that needs resolving regarding this line ?